vignettes/wpgsd_corr_example.Rmd

```diff
@@ -66,197 +70,90 @@ event_tbl %>%
 The the corrleation could be simply calculated as 
 $$Corr(Z_{11},Z_{21})=\frac{80}{\sqrt{100*110}}=0.76$$
 ```{r}
-Corr1 <- 80 / sqrt(100 * 110)
-round(Corr1, 2)
+Corr1=80/sqrt(100*110)
```

Comment:
Please use `<-` instead of `=`, for consistency with the entire package. Please check the entire Rmd file for the usage of `=`.